adding-behavior/changing-the-dom.md

```diff
@@ -0,0 +1,255 @@
+# Changing the DOM
+
+## Goals
+
+The goal for this lesson is to display one of JavaScript's strengths: accessing and changing the DOM.
+
+The skills of changing the DOM using vanilla JavaScript will transfer to other front-end skills, such as working with React.
+
+## Introduction
+
+Keira is a crab enthusiasm, and her enthusiasm for crustaceans goes as deep as the ocean. She's making a fan site for crabs. She's already made a draft website for it.
+
+However, she's in a very strange situation-- she knows what her `index.html` file will look like when it loads... and she wants to change how it will appear _after_ it loads. Keira needs to change the content and appearance of her site _after_ it loads in the browser, _without_ changing the HTML and CSS.
```

Comment:
assorted minor fixes (most important being enthusiasm → enthusiast)
```suggestion
Keira is a crab enthusiast, and her enthusiasm for crustaceans goes as deep as the ocean. She's making a fan site for crabs. She's already made a draft website for it.

However, she's in a very strange situation—she knows what her `index.html` file will look like when it loads... but she wants to change how it will appear _after_ it loads. Keira needs to change the content and appearance of her site _after_ it loads in the browser, _without_ changing the HTML and CSS.
```